pyaerocom/stationdata.py

```diff
@@ -56,7 +56,8 @@ class StationData(StationMetaData):
 
     #: maximum numerical distance between coordinates associated with this
     #: station
-    _COORD_MAX_VAR = 0.1 #km
+    # _COORD_MAX_VAR = 0.1 #km
+    _COORD_MAX_VAR = 1.0 #km
```

Comment:
This is just a little relaxation that helped a little to read several years of Airnow and MarcoPolo data